delivery_carrier_account/models/carrier_account.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2014-TODAY Florian da Costa Akretion <http://www.akretion.com>.
+# Copyright 2014 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class CarrierAccount(models.Model):
+    _name = "carrier.account"
+    _description = "Base account datas"
+    _order = "sequence"
+
+    @api.model
+    def _selection_file_format(self):
+        """To inherit to add label file types"""
+        return [("PDF", "PDF"), ("ZPL", "ZPL"), ("XML", "XML")]
+
+    name = fields.Char(required=True)
+    sequence = fields.Integer()
+    delivery_type = fields.Selection(
```

Comment:
Is this working when a new module is installed? I have also doubts about the translations. I use this method:

https://github.com/OCA/sale-workflow/blob/6dd6786ce468f5ef33fa8e11cfd842ac5223ddee/sale_order_type/models/sale_order_type.py#L17

Reply:
You are right, the translation are not working this way... I've updated following your example, thanks